src/Uno.UI/UI/Xaml/Media/AcrylicBrush/Android/CustomRealtimeBlurView.Android.cs

```diff
@@ -0,0 +1,428 @@
+﻿using System;
```

Comment:
Is this coming from the same author ?

Reply:
Yes, I used this copy for testing and removing things that were not relevant, I will still need to go through it and clean up, but will restore the header copyright

Reply:
Added links to sources to headers of the source code files, is it ok like this?